examples/generic-node-example/src/types/ColorsAbi__factory.ts

```diff
@@ -0,0 +1,226 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
please do not use tslint disable and eslint disable

Reply:
this is a autogenerated file after using typechain